README.md

```diff
@@ -21,6 +21,7 @@ exposing to the internet, because
  - `GitHub` push events (and ping events)
  - `DockerHub` image push events
  - `GitLab` push events
+ - `Bitbucket` push events
```

Comment:
Suggestion: mention both bitbucket cloud (which I failed to do in #13) and bitbucket server.

Reply:
I'll do this in a subsequent PR, to save you some faff.